frontend/src/components/common/ForestClientCard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div>
+        <b-card class="mb-2">
+            <b-card-text>
+                <Tag
+                    :text="props.status.description"
+                    :active="props.status.status_code === 'A' ? true : false"
```

Comment:
Doesn't the `===` comparison already return you `true : false`?

Reply:
Thanks, I will fix it